test/test_contraction_sequence.jl

```diff
@@ -49,9 +49,9 @@ using Test: @test, @testset
       Pkg.rm("KaHyPar"; io=devnull)
       res_kahypar_bipartite = contract(tn; sequence=seq_kahypar_bipartite)[]
       @test res_optimal ≈ res_kahypar_bipartite
-      seq_einexprs_kahypar = contraction_sequence(tn; alg="einexpr", optimizer=HyPar())
-      res_einexprs_kahypar = contract(tn; sequence=seq_einexprs_kahypar)[]
-      @test res_einexprs_kahypar ≈ res_optimal
+      # seq_einexprs_kahypar = contraction_sequence(tn; alg="einexpr", optimizer=HyPar())
+      # res_einexprs_kahypar = contract(tn; sequence=seq_einexprs_kahypar)[]
+      # @test res_einexprs_kahypar ≈ res_optimal
```

Comment:
Could you add a comment saying these were leading to some some issue on CI that needs to be investigated?

Reply:
Sure